helm/hpcc/templates/network.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: default-deny-all
+spec:
+  podSelector: {}
+  policyTypes:
+  - Ingress
+  - Egress
+  egress:
+  - to:
+#  This only works if we know that the kube-system namespace has been labelled, which we cannot control?
+    - namespaceSelector:
+        matchLabels:
+          name: kube-system
```

Comment:
so if the name space of the kube-system is not "kube-system" this NP will not be in force?
In other words, if name space doesn't match - this deny-all NP is not used?

Reply:
This isn't a deny - it's an allow... meaning if it is not applied no-one can call kubectl

And it's not the name of the namespace that matters but its label, which is not present by default.